test/relatedSpec.test.js

```diff
@@ -11,5 +11,27 @@ describe('FEC - Omozan Related/Outfit tests', function () {
     render(<App/>)
   })
 
-  // test('')
+  test('it should render related carousel', () => {
+
+    return waitFor(()=> expect(screen.queryByText(/RELATED PRODUCTS/)).toBeInTheDocument())
+    .then(()=> expect(screen.getByTestId('relatedCarousel').children.length).toBeGreaterThan(0))
+  })
```

Comment:
I could be misreading what you are intending here, but 'related products' should be rendered immediately. Which means that it won't actually know how many children will be in the carousel unless you are rendering dummy data before the API request returns the data. I believe this would mean that the length of the children will never be greater than 0.

Reply:
thanks, will correct.